tracer/champsim_tracer.cpp

```diff
@@ -14,6 +14,8 @@
 #define NUM_INSTR_DESTINATIONS 2
 #define NUM_INSTR_SOURCES 4
 
+using namespace std;
```

Comment:
This is also useful in getting tracer working on newer PIN tool versions!

Reply:
Great! Thank you for confirming this.